src/cmdline/mod.rs

```diff
@@ -12,7 +12,7 @@ use std::fmt;
 use std::result;
 
 /// The error type for command line building operations.
-#[derive(PartialEq, Debug)]
+#[cfg_attr(test, derive(Debug, PartialEq))]
```

Comment:
Any reason for not having `Debug` and `PartialEq` in release build as well?

Reply:
Good catch. These are leftovers from playing with `kcov` runs. I'll clean them up.
They will likely be needed when this crate is consumed by a vmm.